exp/api/v1beta1/gcpmanagedcontrolplane_types.go

```diff
@@ -148,6 +148,16 @@ type GCPManagedControlPlaneSpec struct {
 	// This feature is disabled if this field is not specified.
 	// +optional
 	MasterAuthorizedNetworksConfig *MasterAuthorizedNetworksConfig `json:"master_authorized_networks_config,omitempty"`
+	// LoggingService represents configuration of logging service feature of the GKE cluster.
+	// Possible values: none, logging.googleapis.com/kubernetes (default for GKE 1.14+), logging.googleapis.com (default for GKE < 1.14).
```

Comment:
Should we have some validation for this? Or shall we assume it's a cluster greater than 1.14 and so the only values allowed would be:
- none
- logging.googleapis.com/kubernetes

Reply:
IIUC the google [documentation](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#resource:-cluster) we can't use `logging.googleapis.com` for GKE > 1.14 so yes it makes sense. I'll update the PR.
Thanks.